components/automate-deployment/pkg/bootstrap/bootstrap.go

```diff
@@ -62,6 +64,20 @@ func FullBootstrap(ctx context.Context,
 		return err
 	}
 
+	if bootstrapBundlePath != "" {
```

Comment:
My only concern with doing this here, is that the point of some of this indirection was to protect us against version/code mis-matches during deployment.  That is, the rest of this code is shelling out to the deployment-service command to handle the bootstrapping.  The thought was that we might want to be able to deploy a version of the deployment-service that doesn't line up with the version of automate-cli exactly.

I'm not sure if we really care about that anymore though.

Reply:
we can probably move it into `DeployDeploymentService`. I'm not sure I see where we're calling the deployment service binary. These functions seem to just use local target

Reply:
https://github.com/chef/automate/blob/8349e23d64d92d53cd1f53f766d81edd29377955/components/automate-deployment/pkg/bootstrap/bootstrap.go#L113-L116

Reply:
Once we have hab installed, we turn everything else over to the dsCmd if possible.  As I said above, I'm not sure we want to do this still, but just wanted to point it out in case we do.

Also, I think we should probably make a specific Bootstrapper function for this and call that from FullBootstrap rather than doing it inline, just to remain consistent.

Reply:
i see